platform-sdk/swirlds-base/src/main/java/com/swirlds/base/internal/impl/BaseExecutorThreadFactory.java

```diff
@@ -27,10 +27,19 @@
  */
 class BaseExecutorThreadFactory implements ThreadFactory {
 
+    /**
+     * The singleton instance of this factory.
+     */
     private static final BaseExecutorThreadFactory instance = new BaseExecutorThreadFactory();
```

Comment:
Should we have a ticket to convert this to have an inner InstanceHolder class?

Reply:
https://github.com/hashgraph/hedera-services/issues/12869